_posts/2015-09-24-agile-fluency.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "ePages turns agile"
+date: "2015-09-24 10:10:00"
+icon: refresh
+categories: agile
+authors: ["Anja B."]
+---
+
+Did you ever wonder how companies that are going the agile way are developing further over time?
```

Comment:
Have you ever wonder how companies that are going the agile way are developing further over time?